### DISCUSSION TITLE
When the taskbar is set to the right, the location of the start window is located above the taskbar.

### DISCUSSION THREAD

**Discussion by heonny** [valinet/ExplorerPatcher#3588] [Q&A]
![image](https://github.com/user-attachments/assets/082bf8e2-fbb8-43e7-9c04-d81b89bcc45a)

As you can see in the picture, when the taskbar is set to the right, you can see that the start window invades the location of the taskbar.

My version is `23H2 KB5041655`, and the version of Explorer Patcher is `22621.3527.65.5`.

![image](https://github.com/user-attachments/assets/aa29ab92-8760-49c8-9d3d-43eb87e154b4)

Also, when the start window is placed like this, the account pop-up is abnormally turned on, as shown in the second picture.

**Comment by pyrates999** [comment]
Install the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases

I think I remember seeing this already fixed in that version.

**Reply by babychef** [reply]
How do I download the pre-release versions?

**Reply by pyrates999** [reply]
Look for the label pre-release and expand assets.  Download ep_setup.exe if you're running x86-64, or ep_setup_arm64.exe if you're running arm.  Read the instructions there as you need to exclude some directories with your AV.
